kiota_serialization_json/json_parse_node.py

```diff
@@ -291,6 +293,10 @@ def _assign_field_values(self, item: U) -> None:
                     deserialize but the model doesn't support additional data"
                 )
 
+    def is_four_digit_number(self, value: str) -> bool:
```

Comment:
just making sure this method is not accessible from outside of the class?

Reply:
It's an instance method, and we really just need it here in the entire project, so nope, not accessible outside the class

Reply:
shouldn't it start with __ ?